docs/sdk-guide.md

```diff
@@ -709,37 +749,7 @@ Find example code at [CarbonReaderExample](https://github.com/apache/carbondata/
   public <T> CarbonReader<T> build();
 ```
 ### Class org.apache.carbondata.sdk.file.CarbonSchemaReader
```

Comment:
it is better to mark as deprecated and remove in next version

Reply:
ok, done